### DISCUSSION TITLE
ITV [SN]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#447] [Show and tell]
### Channel ID

ITV.sn

### Stream URL

http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x82er14

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x82er14 (offline)

**Comment by ghost** [comment]
completely down
